doc/nrf/test_and_optimize/optimizing/power_general.rst

```diff
@@ -181,11 +181,33 @@ Bluetooth Mesh
 
 The Bluetooth Mesh protocol offers the :ref:`ug_bt_mesh_configuring_lpn` feature for optimizing the power consumption of the Bluetooth Mesh devices.
 
+Gazell
+======
+
+Gazell is an asymmetrical protocol where the Device role is power-optimized.
+The exact current consumption depends on a number of parameters that are defined statically in the :ref:`nrfxlib:gzll_constants`.
```

Comment:
```suggestion
The exact current consumption depends on a number of parameters that are defined statically in the API.
See the Constants section of the :ref:`Gazell Link Layer API documentation <nrfxlib:gzll_api>`.
```
This link failed the doc build. Now, it should be ok. Feel free to change the wording if this proposed text does not work.

Reply:
Fixed.